### DISCUSSION TITLE
version 2.0.1 problem

### DISCUSSION THREAD

**Discussion by elenaaralla** [SSShooter/mind-elixir-core#169] [General]
Hi,

with version 2.0.1 the map has some "style"problem; 
colored node aren't fully colored (only text background)

with this versione, the plugin export-image, that worked fine with version 1.1.5, doesn't work anymore:
I have this javascript error in console:  
```
port-image.js:5475 Uncaught TypeError: Cannot read properties of null (reading 'getBoundingClientRect')
    at export-image.js:5475:58
    at Zv (export-image.js:5466:37)
    at pe.tm [as exportSvg] (export-image.js:5529:11)
    at exportMap ((index):456:6)
    at HTMLButtonElement.onclick ((index):137:74)
```

I know that it is WIP, but it worked :(

The node-menu doesn't appear; I've see on change log that files that implemented it are deleted; is it possible to restore this functionality?

Thank you,
Elena

**Comment by SSShooter** [comment]
Sorry, export-image is not compatible with 2.x yet 😭
For node-menu, I had moved it to [@mind-elixir/node-menu](https://github.com/ssshooter/node-menu), so you can easily customize this plugin.
And……I don't really understand the meaning of `fully colored`?

**Reply by elenaaralla** [reply]
Hi,

thank you; 

I've attached two images to explain "fully colored" :)

have a nice day,Elena.


![Schermata 2023-05-05 alle 07 49 27](https://user-images.githubusercontent.com/14870650/236385931-f4f88d63-7a06-4769-af29-82c1b6fff76f.jpg)
![Schermata 2023-05-05 alle 07 49 56](https://user-images.githubusercontent.com/14870650/236385940-d3521a23-7e86-40ca-a9a2-7fc499e6a147.jpg)

**Reply by SSShooter** [reply]
OK, I may fix it this week.

**Reply by elenaaralla** [reply]
thank you!

**Reply by SSShooter** [reply]
Fixed in 2.0.2

**Reply by elenaaralla** [reply]
Thank you @ssshooter!

It works for alla node, except for the first (main) node :)

Have a nice day.

![Schermata 2023-06-01 alle 15 24 25](https://github.com/ssshooter/mind-elixir-core/assets/14870650/a3d55eeb-c8e8-41e1-9b1e-44cd3ec58e0d)
